### PR TITLE
Added ORT Web info to HF landing page

### PR DESCRIPTION
**Pull request by sophies927**
Updated w/ Transformers.js info

### CODE REVIEW

**Review by MaanavD — APPROVED** [review]
LGTM. Looked at preview! Simple changes. 🚀

File: src/routes/huggingface/+page.svelte [hunk 1]
```diff
@@ -44,8 +44,8 @@
 		<div class="col-span-2">
 			<h1 class="text-3xl pb-4">Export Hugging Face Models to ONNX</h1>
 			<p class="pb-4">
-				Hugging Face provides many options for exporting models to ONNX, including an ONNX Export Space
-				for PyTorch models from the Hugging Face Model Hub.
+				Hugging Face provides many options for exporting models to ONNX, including an ONNX Export
+				Space for PyTorch models from the Hugging Face Model Hub.
```
Comment (natke) [inline src/routes/huggingface/+page.svelte:48]: What does ONNX export space mean?

File: src/routes/huggingface/+page.svelte [hunk 2]
```diff
@@ -135,14 +135,37 @@
...
+<div class="container mx-auto px-10 my-10">
+	<div class="divider" />
+	<div>
+		<div>
+			<h1 class="text-3xl pb-4">Transformers.js + ONNX Runtime Web</h1>
+			<p class="pb-4">
+				<a href="https://huggingface.co/docs/transformers.js/index" class="text-blue-500"
+					>Transformers.js</a
+				>
+				is amazing tool to run transformers on the web, designed to be functionally equivalent to Hugging
```
Comment (natke) [inline src/routes/huggingface/+page.svelte:159]: an amazing tool

File: src/routes/huggingface/+page.svelte [hunk 3]
```diff
@@ -135,14 +135,37 @@
...
 			</p>
 			<p>
-				<a href="https://ml.azure.com/" class="text-blue-500">Azure Machine Learning</a> also publishes a
-				curated model list that is updated regularly and includes some of the most popular models at the
-				moment. Of the models on this list that are available on Hugging Face, there is currently Optimum
-				ONNX support for over 85%.
+				<a href="https://ml.azure.com/" class="text-blue-500">Azure Machine Learning</a> also publishes
+				a curated model list that is updated regularly and includes some of the most popular models at
+				the moment. Of the models on this list that are available on Hugging Face, there is currently
+				Optimum ONNX support for over 85%.
```
Comment (natke) [inline src/routes/huggingface/+page.svelte:141]:
```suggestion
				<a href="https://ml.azure.com/" class="text-blue-500">Azure Machine Learning</a> publishes
				a curated model list that is updated regularly and includes the most popular models. You can run 
				the vast majority of the models on the curated list with ONNX Runtime, using HuggingFace
				Optimum.
```

**Review by natke — APPROVED** [review]
A few editorial comments. Take or leave :)

**Pull request merged**